### PR TITLE
Upgrade docs theme

### PR DESCRIPTION
**Pull request by shuding**
This is an example of using the built-in Nextra theme to extend the blog layout. Lots of wrong styling to migrate still.

https://turbo-site-a1jnp3een.vercel.sh

### CODE REVIEW

**Comment by jaredpalmer** [comment]
@shuding what additional steps do I need to take to push this through goal line

**Comment by jaredpalmer** [comment]
Made some homepage tweaks here: https://github.com/vercel/turborepo/pull/931

**Comment by jaredpalmer** [comment]
Done

File: docs/theme.config.js [hunk 1]
```diff
@@ -118,66 +119,20 @@ const Logo = ({ height, ...props }) => (
...
-  },
-  gsoltis: {
-    name: "Greg Soltis",
-    twitterUsername: "gsoltis",
-    picture: "/images/people/gsoltis.jpeg",
-  },
-};
-
 const theme = {
   github: "https://github.com/vercel/turborepo",
```
Comment (jaredpalmer) [inline docs/theme.config.js:123]: The github link in the nav goes to nextra right now

**Review by jaredpalmer — CHANGES_REQUESTED** [review]

**Comment by shuding** [comment]
Missing x-padding here:

508DC39C-CEF9-46CF-9E54-1E49FBD60447

**Pull request merged**